src/hacbs/components/PipelineRunDetailsView/visualization/edges/PipelineRunEdge.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/Topology/topology-components';
+import { Edge } from '@patternfly/react-topology';
+import { observer } from 'mobx-react';
+import { integralShapePath } from '../../../ApplicationDetails/tabs/overview/visualization/utils/draw-utils';
+import { DEFAULT_EDGE_INDENT } from '../../../topology/const';
+
+interface PipelineRunEdgeProps {
+  element: Edge;
+  className?: string;
+}
+
+const PipelineRunEdge: React.FunctionComponent<PipelineRunEdgeProps> = ({ element, className }) => {
```

Comment:
We should update the PF topology `TaskEdge` to handle the end indent and reuse it.

Reply:
Yes, I will log an issue on PF for this usecase.